packages/expo-constants/src/ExponentConstants.web.ts

```diff
@@ -1,19 +1,28 @@
 import UAParser from 'ua-parser-js';
-import uuidv4 from 'uuid/v4';
+import UUID from 'uuid-js';
+
 import { PlatformManifest, WebManifest, NativeConstants } from './Constants.types';
 
-const ExpoPackageJson = require('expo/package.json');
+function getExpoVersion(): string | null {
+  try {
+    // Remove the need to install the entire expo package.
```

Comment:
I think that needing to install the expo package is fine -- what projects would use expo-constants without expo? A bigger issue seems to be that `expo/package.json` is a large file and all we need is the "version" field. Ideally, webpack would be able to pluck out just the version field -- it's basically tree shaking -- but I doubt it supports that. We could have a tiny export in version.js that just exports a constant, and a unit test that ensures package.json and version.js are synced.